deno/payloads/v9/channel.ts

```diff
@@ -344,6 +344,10 @@ export interface APIMessage {
 	author: APIUser;
 	/**
 	 * Contents of the message
+	 *
+	 * The `MESSAGE_CONTENT` (`1 << 15`) privileged gateway intent will become required after August 31, 2022 for verified applications, to receive a non-empty value from this field
```

Comment:
Isn't this applicable only to API v10?

Reply:
currently its only enforced on v10, after the deadline it will be enforced on all API versions

however for REST it's pointless to put the bit value, since it only depends on the dev portal toggle

Reply:
(From what I understood) API v10 needs to put the bit value when connecting to the gateway, and have the toggle from Discord Developers Portal enabled, but in API v9, **only the toggle** needs to be enabled. Is it correct?

Reply:
oh this is in payloads; yeah i think that's right

Reply:
Oh, right. Maybe I can change the descriptions with this information for both API versions, so as to not generalize.
In API v10 can be like:

> The **`MESSAGE_CONTENT`** privileged gateway intent is required for verified applications to receive a non-empty value from this field. In Discord Developers Portal, you need to enable the toggle of this intent of your application in **Bot > Privileged Gateway Intents**. You also need to specify the intent bit value  (**`1 << 15`**) when connecting to the gateway.

and for API v9 can be like:

> The **`MESSAGE_CONTENT`** privileged gateway intent will become required after **August 31, 2022** for verified applications to receive a non-empty value from this field. In Discord Developers Portal, you need to enable the toggle of this intent of your application in **Bot > Privileged Gateway Intents**.

Reply:
> > The **`MESSAGE_CONTENT`** privileged gateway intent is required for verified applications to receive a non-empty value from this field. In Discord Developers Portal, you need to enable the toggle of this intent of your application in **Bot > Privileged Gateway Intents**. You also need to specify the intent bit value (**`1 << 15`**) when connecting to the gateway.

I'd correct this to `You also need to specify the intent bit value...if you are connecting to the gateway`